### PR TITLE
Add HA job to the Cloud 6 Gating

### PR DESCRIPTION
**Pull request by dirkmueller**
Now that the HA job is finally reliably green, lets add
it to the gating in order ot not break it again.

### CODE REVIEW

**Comment by bmwiedemann** [comment]
+1

**Comment by toabctl** [comment]
+1

**Comment by aspiers** [comment]
+1 for the idea, although is there no way to avoid the duplication of yaml from cloud-mkcloud6-job-ha.yaml?

**Comment by bmwiedemann** [comment]
maybe we could trigger the ha-trigger instead of mkcloud directly.

**Comment by dirkmueller** [comment]
@bmwiedemann @aspiers last time I tried that it didn't work.

**Comment by toabctl** [comment]
there are job templates and macros. Should be possible

File: scripts/jenkins/jobs-ibs/cloud-mkcloud6-gating.yaml [hunk 1]
```diff
@@ -36,11 +37,21 @@
             - name: openstack-mkcloud
               node-label: cloud-trigger
               predefined-parameters: |
+                TESTHEAD=1
                 cloudsource=develcloud6
                 nodenumber=4
                 want_ldap=1
                 networkingplugin=linuxbridge
                 mkcloudtarget=all
+            - name: openstack-mkcloud
```
Comment (bmwiedemann) [inline scripts/jenkins/jobs-ibs/cloud-mkcloud6-gating.yaml]: we could try to make this
name: cloud-mkcloud6-job-ha
and drop all predefined-parameters below.

**Comment by dirkmueller** [comment]
the weather report was wrong last time I trie.d but I can try again.

File: scripts/jenkins/jobs-ibs/cloud-mkcloud6-gating.yaml (hunk 1 above)
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/cloud-mkcloud6-gating.yaml]: As I commented before, it doesn#t work for the weather report from what I remember.

**Comment by bmwiedemann** [comment]
+1 if that works.

**Comment by dirkmueller** [comment]
conflicts with the templatizing in https://github.com/SUSE-Cloud/automation/pull/892

**Comment by rsalevsky** [comment]
Needs rebase

**Comment by vuntz** [comment]
> Needs rebase

No, there's no conflict and can be merged by admins.

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-gating-template.yaml [hunk 2]
```diff
@@ -36,11 +37,18 @@
...
                 cloudsource=develcloud{version}
                 nodenumber=4
                 want_ldap=1
                 networkingplugin=linuxbridge
                 mkcloudtarget=all
+      - multijob:
+          name: 'Extra Gate Checks (HA)'
+          condition: SUCCESSFUL
+          projects:
+            - name: cloud-mkcloud6-job-ha
```
Comment (rsalevsky) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-gating-template.yaml]: I think this should be 'cloud-mkcloud{version}-job-ha'.

**Comment by rsalevsky** [comment]
Why got this PR merged? There was still one open question.

**Pull request merged**